### PR TITLE
feat(switch): add support for Switch

### LINKED ISSUE #21: Add support for Switch
Hello,

I would like to identify card from Switch. You can get Switch traits on [Wikipedia](https://en.wikipedia.org/wiki/Payment_card_number#Issuer_identification_number_%28IIN%29).

If you want to solve this issue, check this [how-to guide](https://github.com/d-edge/Cardidy/blob/main/CONTRIBUTING.md#add-a-new-card).

Cheers,

### PR DESCRIPTION
**Pull request by ntbm**
Closes #21

### CODE REVIEW

**Comment by aloisdg** [comment]
The tests are failing. Could you fix them?

```
A total of 1 test files matched the specified pattern.
  Failed ShouldIdentifyAsMaestros("6759710000901011") [66 ms]
  Error Message:
     Expected is <DEdge.CardType[3]>, actual is <System.Linq.Enumerable+WhereSelectArrayIterator`2[DEdge.Model.ICard,DEdge.CardType]>
  Values differ at index [2]
  Expected: GPN
  But was:  Switch
```

source: https://github.com/d-edge/Cardidy/actions/runs/3164565462/jobs/5156547997#step:6:1

**Comment by ntbm** [comment]
Repushed a signed version of the code.
Also fixed the tests.
Sorry for not running them before and also thank you for providing me an opportunity to learn about payment cards :)

**Comment by aloisdg** [comment]
You are welcome. Thanks for you work!

File: src/DEdge.Cardidy/Model/Cards.cs [hunk 1]
```diff
@@ -100,6 +100,11 @@ internal record Laser : ALuhnCard
     public Laser() : base(CardType.Laser, new[] { 6304, 6706, 6771, 6709 }, From16To19) { }
 }
 
+internal record Switch : ALuhnCard
+{
+    public Switch() : base(CardType.Switch, new []{4903, 4905, 4911, 4936, 564182, 633110, 6333, 6759}, From16To19) { }
```
[the remark this answers is not in the document]
Comment (aloisdg) [reply src/DEdge.Cardidy/Model/Cards.cs:105]: @ntbm up ;)

**Review by aloisdg — CHANGES_REQUESTED** [review]

**Review by torendil — APPROVED** [review]
Hey! LGTM :)

**Review by aloisdg — APPROVED** [review]

**Pull request merged**